src/App.js

```diff
@@ -1,23 +1,40 @@
-import logo from './logo.svg';
+import axios from 'axios';
+import { Route, Routes } from 'react-router-dom';
+import React from 'react';
 import './App.css';
+import Header from './components/Header';
+import HomePage from './components/HomePage';
+import Beers from './components/Beers';
+import BeerDetails from './components/BeerDetails';
+import { useEffect, useState } from 'react/cjs/react.production.min';
+import CreateBeer from './components/CreateBeer';
 
 function App() {
+
+  const [beers, setBeer] = useState([]);
+
+  useEffect(() => {
+    axios.get("https://ih-beers-api2.herokuapp.com/beers")
+      .then((response) => {
+        //console.log(response.data);
+        setBeer(response.data);
+      })
+      .catch((e) => console.log("error fetchin array of beers", e));
+  }, []);
+
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+<div className='App'>
+    <Routes>
+      <Route path="/" element={<HomePage />} />
```

Comment:
`<Header>` is always there. So it's not a Route, you put it outside of the Routes.
Changed your code a bit, as an example. 
Not sure why you have `/create` and `/newbeer`, so what's the difference and `id` must be dynamic and not `1.`
The add button needs to be inside the CreateBeer component.
```
  <Header />
      <Routes>
        <Route path="/" element={<HomePage />} />
        <Route path="/beers" element={<Beers />} />
        <Route path="/random-beer" element={<></>} />
        <Route path="/new-beer" element={<CreateBeer />} />
        <Route path="/beers/1" element={<BeerDetails beerArr={beers} />} />
        <Route path="/create" element={<CreateBeer />} />
      </Routes>
```